lab1.js

```diff
@@ -94,7 +123,9 @@ var nextAnimal;
 // TODO: 10 points
 // Assign one of your favorite animals to nextAnimal using Math.random() to pick
 
-// your code goes here
+var bestAnimal = favoriteAnimals.length;
+var animal = Math.floor((Math.random() * bestAnimal) + 1);
```

Comment:
same error here as in the meerkat problem. you end up sometimes trying to index 
favoriteAnimals[4]
which isn't going to work. That's why this assertion only fails sometimes.